packages/Popover/src/Popover.story.tsx

```diff
@@ -55,8 +55,16 @@ function DefaultExample() {
     >
       {text('Button Content', 'Popover')}
       <Popover
-        align={select('Align', ['top', 'right', 'bottom', 'left'], 'top')}
-        justify={select('justify', ['start', 'middle', 'end'], 'start')}
+        align={select(
+          'Align',
+          ['top', 'right', 'bottom', 'left'] as const,
```

Comment:
Can we import the object and do Object.values? Ditto everywhere else?

Reply:
⬆️